app/controllers/introduce_yourself_controller.rb

```diff
@@ -17,6 +17,10 @@ def update
 
   private
 
+  def ensure_application_present
+    _application_not_created = true
+  end
```

Comment:
it would be possible to go `back` and get here with a current snap app, so maybe it doesn't need its own implementation?